Cairo Desktop/Cairo Desktop/Services/DesktopManager.cs

```diff
@@ -96,14 +96,17 @@ private void InitDesktop()
             if (!IsEnabled && !EnvironmentHelper.IsAppRunningAsShell)
                 return;
 
-            // hide the windows desktop
-            ShellHelper.ToggleDesktopIcons(false);
-
+            HideWindowsDesktopIcons();
             CreateShellWindow();
             CreateDesktopBrowser();
             CreateDesktopWindow();
         }
 
+        private static void HideWindowsDesktopIcons()
+        {
+            if (IsEnabled && !EnvironmentHelper.IsAppRunningAsShell) ShellHelper.ToggleDesktopIcons(false);
```

Comment:
Is there a reason why `!EnvironmentHelper.IsAppRunningAsShell` is part of this condition, given that `ShellHelper.ToggleDesktopIcons` already checks this?

Reply:
Didn't even notice it did that